assets/components/button/button.scss

```diff
@@ -7,6 +7,7 @@
   text-decoration: none;
   font-weight: bold;
   height: $gu-cta-height;
+  min-height: $gu-cta-height;
```

Comment:
Thanks for explaining why you needed to do this @jranks123. Maybe just do a smoke test on a few random other pages on the support site.